packages/page-accounts/src/Sidebar/AddressSection.tsx

```diff
@@ -46,15 +61,36 @@ function AddressSection ({ accountIndex, defaultValue, editingName, flags, onCha
           value={value}
           withSidebar={false}
         />
+       
         <div className='ui--AddressMenu-addr'>
           {value}
         </div>
+        
         {accountIndex && (
           <div className='ui--AddressMenu-index'>
             <label>{t<string>('index')}:</label> {accountIndex}
           </div>
         )}
       </div>
+
+      <div className='ui--AddressSection__CopyColumn'>
+        <div className='ui--AddressMenu-copyaddr'>         
+          <CopyToClipboard text={value}
+          onCopy={() => setCountCopied(countCopied + 1)}
```

Comment:
Rather use a callback. There is also a `useIncrement` hook that wraps inc/dec already, see https://github.com/polkadot-js/apps/blob/master/packages/react-hooks/src/useIncrement.ts